apps/prairielearn/src/ee/pages/instructorAiGenerateJobDetails/instructorAiGenerateJobDetails.html.ts

```diff
@@ -0,0 +1,79 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { HeadContents } from '../../../components/HeadContents.html.js';
+import { Job } from '../../../lib/db-types.js';
+
+export function AiGenerateJobDetailsPage({
+  resLocals,
+  job,
+}: {
+  resLocals: Record<string, any>;
+  job: Job;
+}) {
+  return html` <!doctype html>
```

Comment:
Conventionally we wrap this onto a new line.

```suggestion
  return html`
    <!doctype html>
```

Reply:
changed in 3c8bafd1470728a8c79b488213de1dfae0aed79e